Source/WebCore/page/Quirks.cpp

```diff
@@ -496,11 +496,9 @@ bool Quirks::shouldDispatchSimulatedMouseEvents(const EventTarget* target) const
         return false;
 
     case ShouldDispatchSimulatedMouseEvents::DependingOnTargetFor_mybinder_org:
-        if (is<Node>(target)) {
-            for (const auto* node = downcast<Node>(target); node; node = node->parentNode()) {
-                if (is<Element>(node) && const_cast<Element&>(downcast<Element>(*node)).classList().contains("lm-DockPanel-tabBar"_s))
-                    return true;
-            }
+        for (RefPtr node = dynamicDowncast<Node>(target); node; node = node->parentNode()) {
+            if (RefPtr element = dynamicDowncast<Element>(node); element && const_cast<Element&>(element).classList().contains("lm-DockPanel-tabBar"_s))
```

Comment:
I also tried this without const_cast<Element&>() and that didn't compile either on Linux. @mcatanzaro do you perhaps have advice for me? Thanks!

Reply:
So you have a `RefPtr<const Element>` (unfortunate!) and you want to get a non-const `Element` from it? You're surely not going to be able to cast from `RefPtr` directly to `Element` (how does that work? really surprised that builds on Apple platforms). Instead, try getting at the `const Element` first, _then_ do the `const_cast`. I would try: `const_cast<Element*>(element.get())->classList`

Reply:
Thank you for identifying the issue so clearly. Your suggestion didn't work, but removing `const` altogether seems to work fine and it wasn't really buying us anything here.